File: CS/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace WindowsApplication1
{
    public partial class Form1 : Form
    {

        public Form1()
        {
            InitializeComponent();
            repositoryItemMyButtonEdit3.TextToScroll = String.Format("20°C{0}{0}London{0}{0}30°C{0}{0}Paris{0}", Environment.NewLine);
        }


        const int maxValue = 100;
        int value = -maxValue;

        private void timer1_Tick(object sender, EventArgs e)
        {
            value++;
            if (value > maxValue)
                value = -maxValue;
            barEditItem2.EditValue = value;
        }
    }
}

File: VB/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace WindowsApplication1
	Partial Public Class Form1
		Inherits Form

		Public Sub New()
			InitializeComponent()
			repositoryItemMyButtonEdit3.TextToScroll = String.Format("20°C{0}{0}London{0}{0}30°C{0}{0}Paris{0}", Environment.NewLine)
		End Sub


		Private Const maxValue As Integer = 100
		Private value As Integer = -maxValue

		Private Sub timer1_Tick(ByVal sender As Object, ByVal e As EventArgs) Handles timer1.Tick
			value += 1
			If value > maxValue Then
				value = -maxValue
			End If
			barEditItem2.EditValue = value
		End Sub
	End Class
End Namespace